packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue

```diff
@@ -655,10 +654,18 @@ const onDownloadResponse = (payload) => {
 }
 
 .rerun-message {
+	position: sticky;
+	top: 0;
+	z-index: 5;
```

Comment:
the code cells line of code counter seem to have some sort of z-index on them, setting this to 5

Reply:
@YohannParis I put the `isolate` variable.  Works well!